eng/ci/templates/jobs/install-dotnet.yml

```diff
@@ -0,0 +1,20 @@
+jobs:
+
+- job: InstallDotnet
+  displayName: Install Dotnet
+
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-windows-2022
+    os: windows
+
+  steps:
+  - task: PowerShell@2
```

Comment:
There is a dedicated task for dotnet installation: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-dotnet-v2?view=azure-pipelines